tracee/tracee.go

```diff
@@ -341,6 +341,50 @@ func (t *Tracee) processEvent(ctx *context, args []interface{}) error {
 		}
 	}
 
+	//capture executed files
+	if t.config.CaptureFiles && (eventName == "security_bprm_check") {
```

Comment:
Do you think its a good idea to use the same capture files flag for both vfs_write and executables?

Reply:
I don't think it will be common to want one and not the other, and even if so, it's easy to just ignore the other files (maybe in the future we will "capture" more files). If you think otherwise let me know. I generally would prefer to simplify the flags rather then have every new feature under a new flag.